lib/AST/Module.cpp

```diff
@@ -1731,13 +1733,35 @@ static ProtocolConformanceRef getBuiltinTupleTypeConformance(
   return ProtocolConformanceRef::forMissingOrInvalid(type, protocol);
 }
 
+using EitherFunctionType =
+    llvm::PointerUnion<const SILFunctionType *, const FunctionType *>;
+
 /// Whether the given function type conforms to Sendable.
-static bool isSendableFunctionType(const FunctionType *functionType) {
-  if (functionType->isSendable())
-    return true;
+static bool isSendableFunctionType(EitherFunctionType eitherFnTy) {
```

Comment:
This should just take an ExtInfo

Reply:
actually, we should not ever be looking up conformance of SILFunctionType to Sendable. It's a bug if that happens